lib/cli_executor.rb

```diff
@@ -60,7 +60,7 @@ def self.get_version_for(user, host)
       fake_config = Tempfile.new("kubeconfig")
       fake_config.close
 
-      res = host.exec_as(user, "oc version -o yaml --config=#{fake_config.path}")
+      res = host.exec_as(user, "oc version -o yaml --config=#{fake_config.path}" 2>/dev/null)
```

Comment:
Here better to update with --kubeconfig, since it already says Flag --config has been deprecated

Reply:
--kubeconfig comes since oc 3.11 (not 3.10), so safe to update 4.x master branch